temba/contacts/templatetags/contacts.py

```diff
@@ -199,7 +199,7 @@ def history_class(item):
     if isinstance(item, Msg):
         if item.media and item.media[:6] == 'video:':
             css = '%s %s' % (css, 'video')
-        if item.direction or item.recipient_count:
+        if item.direction or (item.broadcast and item.broadcast.recipient_count > 1):
```

Comment:
Ah, then I suspect this [recent change](https://github.com/rapidpro/rapidpro/commit/e6fc0ade5f442ccd88752b3d3cf5f9510666cf2c) is the reason. That creates an empty inbound message when starting a USSD session, the direction isn't being set there.

That would also explain why you're not seeing this error on your installs.